frontend-new/.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
     es2021: true,
   },
   extends: [
+    'next/core-web-vitals',
```

Comment:
I'm still getting the same warning when building: 
```
build: $ next build
build: info  - Linting and checking validity of types...
build: warn  - The Next.js plugin was not detected in your ESLint configuration. See https://nextjs.org/docs/basic-features/eslint#migrating-existing-config
```
Please check that it has the default plugin added, thanks.

Reply:
worked fine for me on node v16.18.0